app/javascript/react/containers/ParkShowContainer.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+
+class ParkShowContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
I would recommend: 

```
this.state = {
  park: {
    ... //put your attributes here.
  }
}
```

this way you can reassign state without having to micromanage each attribute